var/spack/repos/builtin/packages/premake-core/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PremakeCore(Package):
```

Comment:
Can you convert this to `MakefilePackage` and override the `build` and `install` phases?

Reply:
Done in https://github.com/spack/spack/pull/18807/commits/5479873fcc35a043b860511e5e510f2965130342